<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Detail_widget</class>
 <widget class="QWidget" name="Detail_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="central_widget_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="shadow_layout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QFrame" name="background_frame">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 220, 215)</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QFrame" name="top_frame">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 0, 0);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="title_label">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>14</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white
</string>
              </property>
              <property name="text">
               <string>SISTEMA MANTENIMIENTO</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QFrame" name="buttons_holder_frame">
              <property name="minimumSize">
               <size>
                <width>110</width>
                <height>30</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QToolButton" name="minimize_button">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>0</y>
                 <width>21</width>
                 <height>22</height>
                </rect>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../../icons/minimize-window.png</normaloff>../../icons/minimize-window.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
              <widget class="QToolButton" name="restore_button">
               <property name="geometry">
                <rect>
                 <x>50</x>
                 <y>0</y>
                 <width>21</width>
                 <height>22</height>
                </rect>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../../icons/restore-window.png</normaloff>../../icons/restore-window.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
              <widget class="QToolButton" name="maximize_button">
               <property name="geometry">
                <rect>
                 <x>50</x>
                 <y>0</y>
                 <width>21</width>
                 <height>22</height>
                </rect>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../../icons/maximize-window.png</normaloff>../../icons/maximize-window.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
              <widget class="QToolButton" name="close_button">
               <property name="geometry">
                <rect>
                 <x>90</x>
                 <y>0</y>
                 <width>21</width>
                 <height>22</height>
                </rect>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../../icons/close-window.png</normaloff>../../icons/close-window.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="content_frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QFrame" name="frame">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>150</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="maquina_pic_label">
                 <property name="maximumSize">
                  <size>
                   <width>170</width>
                   <height>170</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_3">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <widget class="QLabel" name="maquine_title_label">
                  <property name="geometry">
                   <rect>
                    <x>60</x>
                    <y>10</y>
                    <width>191</width>
                    <height>16</height>
                   </rect>
                  </property>
                  <property name="font">
                   <font>
                    <family>Rockwell</family>
                    <pointsize>12</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>NOMBRE MAQUINA</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="label">
                  <property name="geometry">
                   <rect>
                    <x>100</x>
                    <y>50</y>
                    <width>151</width>
                    <height>16</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string>Manual de la maquina:</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="maquina_cat_label">
                  <property name="geometry">
                   <rect>
                    <x>240</x>
                    <y>50</y>
                    <width>241</width>
                    <height>16</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="label_2">
                  <property name="geometry">
                   <rect>
                    <x>100</x>
                    <y>90</y>
                    <width>151</width>
                    <height>16</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string>Fecha Mantenimiento:</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="maquina_fecha_label">
                  <property name="geometry">
                   <rect>
                    <x>240</x>
                    <y>90</y>
                    <width>241</width>
                    <height>16</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QWidget" name="layoutWidget">
               <property name="geometry">
                <rect>
                 <x>230</x>
                 <y>18</y>
                 <width>221</width>
                 <height>211</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="font">
                   <font>
                    <family>Rockwell</family>
                    <pointsize>12</pointsize>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>SUBSISTEMAS</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPlainTextEdit" name="sub_text_edit"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
